edubots_surveys.Rnw

```diff
@@ -297,16 +297,11 @@ freq.Discipline %>% mutate(Discipline = ifelse(as.character(Discipline) == "No",
-adapting individuals' pace of learning, who get customized online tutoring outside the classroom. Thus, chatbots can significantly contribute to providing interactive learning experience as well as individual attention \cite{agarwal2020}.
-
-Given the potential benefits that chatbots could bring to the classroom, the focus of the EduBots KA2 European project, the sponsor of this study, is to gather, analyse and disseminate best practice of the use of chatbots in higher education institutions.  This study involves two phases of data collection. The first phase aimed to collect feedback from students about their actual needs \cite{MoraChatbots2021}.  Hence, the objective was to collect information about students’ preferences of tools/applications for chatting and messaging, how they use them in educational context, who do they like to be with in the class messaging groups, and their expectations of the chatbots in terms of assisting them during their learning process. Therefore, two surveys for bachelor and master degree students at the University of Granada (Spain) were developed and answers from more than 250 students were collected.   The key findings of the students surveys reveal that they prefer to use messaging applications, such as Telegram or WhatsApp as they are familiar to them. Students also expressed interest in chatbots that can assist them with organizing their course schedules, help them access their assignment grades and facilitate searching for course’s resources.
-
-The results of the first phase were fed into the second phase of the study, which focussed on teachers’ opinions about using chatbots and messaging applications in classes.  Another survey was developed for this purpose. Responses from 300 teachers were collected and analysed.  This study focuses on discussing the implications and the impact of the development of future technology-enhanced tools and educational institutions’ policies by finding answers to the following research questions:\begin{itemize}
+A chatbot is a program, sometimes developed using Artificial Intelligence techniques, that allows a computer to communicate in a similar way as humans do \cite{Gong2008}, using natural language. Indeed, most of them would be, apparently, close to passing a classic Turing test \cite{ moor2003turing }, since they are able to answer almost any question fluently, and even pose questions in their conversations. The recent advancement in this discipline has played an important role in many fields especially in education and online tutoring \cite{clarizia2018,Smutny2020}. These automatic systems facilitate the delivery of personalized learning by adapting to students’ pace of learning and providing customized online tutoring outside the classroom. Thus, chatbots can significantly contribute to providing interactive learning experiences as well as improving individual attention \cite{agarwal2020}.
```

Comment:
The citation with spaces is going to cause an error. `\cite{ moor2003turing }` should be something like `\cite{moor2003turing}`. I'm going to let it pass and then fix it, but please take it into account going forward